syntax = "proto3";
package dfi;

message DFIAppendBufferResponse {
	uint32 return = 1;
}
